src/Contents/mods/coavinssupport2/media/lua/client/coavinssupport2/CoavinsInclude_WeaponPack.lua

```diff
@@ -405,10 +405,10 @@ CoavinsFirearms.Include('Base.ColtM16Tape_HE',    'M16Rifle')
 CoavinsFirearms.Include('Base.ColtM16Tape_INC',   'M16Rifle')
 
 -- M40A1.txt
--- M40
--- M40A1
--- M40A2
--- M40A3
+CoavinsFirearms.Include('M40','Rifle_308BA')
```

Comment:
The string you use here (`M40`) must be the fully qualified Zomboid item name. All of these mod weapons seem to use the prefix `Base.` for whatever reason, rather than the actual mod id like they're supposed to, so please add the prefix `Base.` to all of the item names (like `Base.M40`) in these calls to `CoavinsFirearms.Include()`.